src/middlewared/middlewared/etc_files/syslogd.py

```diff
@@ -9,6 +9,53 @@
 logger = logging.getLogger(__name__)
 
 
+def generate_syslog_remote_destination(middleware, advanced_config):
+    result = ""
+    if advanced_config["syslogserver"]:
+        if ":" in advanced_config["syslogserver"]:
+            host, port = advanced_config["syslogserver"].split(":")[:2]
```

Comment:
Do you think it's good to discard rest of the values silently ( :2 ) ? How about we use `split/rsplit(':', 1)` ?

Reply:
Or maybe not allow such values which contain more then one instance of `:` ?